### ISSUE TITLE
That player cannot be found

### ISSUE THREAD

**Issue by MisterLolxd2** [PikaMug/Quests#2036]
### What is your Quests version / build number (do *not* say "latest")?

Quests version 4.6.1-b373

### Describe the problem and how to replicate it.

This error does not appear for all Quests. However, if it appears, it is at the time of starting / finishing / leaving the quest and when checking the progress of the quest. where then this progress simply does not appear. This is what happens after upgrading to 4.6.1-b373

https://github.com/PikaMug/Quests/issues/2033

### Any console errors to share?

```shell
[14:41:48 INFO]: That player cannot be found
```

### If applicable, can you provide an example quest from quests.yml?

```yaml
If player starts quests like this: https://pastebin.com/Uqvta36N the "That player cannot be found" doesn't send, but if player starts this quest: https://pastebin.com/JGhrd4zq the message sends.

FULL quest.yml: https://pastebin.com/HyegFu40
```

**Comment by PikaMug** [comment]
Thanks for the report. I was able to replicate this on a 1.8 server only; newer versions of Minecraft do not appear to be affected. At a glance, I can't tell whether this is an issue with the implementation at https://github.com/PikaMug/Quests/blob/main/v1_8_R3/src/main/java/me/blackvein/quests/nms/TitleProvider_v1_8_R3.java or a bug with Minecraft/Bukkit/Spigot/Paper.

You might be able to alleviate this by setting `show-titles: false` in _config.yml_, but titles will no longer appear in-game.

**Comment by MisterLolxd2** [comment]
I'm turn show-titles to false, but "That player cannot be found" still sends.

**Comment by MisterLolxd2** [comment]
The issue persists. The message sends in quests: mining, deliviry, but in pvp not. In addition, the progress of tasks is not shown (mining, delivery).

**Comment by PikaMug** [comment]
Resolved in e7a6456

Try https://ci.codemc.io/job/PikaMug/job/Quests/395/
